Engine/Results/LiveTradingResultHandler.cs

```diff
@@ -328,6 +333,28 @@ public void Update()
                         _nextStatisticsUpdate = utcNow.AddMinutes(1);
                     }
 
+                    if (utcNow > _nextStatusUpdate)
+                    {
+                        var chartComplete = new Dictionary<string, Chart>();
+                        lock (ChartLock)
+                        {
+                            foreach (var chart in Charts)
+                            {
+                                // remove directory pathing characters from chart names
+                                var safeName = chart.Value.Name.Replace('/', '-');
+                                DictionarySafeAdd(chartComplete, safeName, chart.Value.Clone(), "chartComplete");
+                            }
+                        }
+                        StoreStatusFile(
+                            // todo: runtimeStatistics should not have ':' in the key
+                            runtimeStatistics.ToDictionary(pair => pair.Key.TrimEnd(':'), pair => pair.Value),
```

Comment:
Not sure if this is still work in progress, just wondering if the key should have the `:` char replaced or removed in `QCAlgorithm.SetRuntimeStatistic` instead?

The `QCAlgorithm.RuntimeStatistics` dictionary is currently `public` though.